packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -179,6 +180,11 @@ class CupertinoSegmentedControl<T> extends StatefulWidget {
   /// Defaults to the selectedColor at 20% opacity if null.
   final Color pressedColor;
 
+  /// The CupertinoSegmentedControl will be placed inside this padding
```

Comment:
Copied from the docs of `_kHorizontalItemPadding`
```suggestion
  /// Minimum padding from edges of the segmented control to edges of encompassing
  /// widget.
```

Reply:
Sorry what I was talking about the `padding` property in `CupertinoSegmentedControl`. The padding is part of the `CupertinoSegmentedControl` widget but it sounded like it is not.